src/content/docs/journey-migration/migrating-dashboards.mdx

```diff
@@ -17,9 +17,9 @@ import journeyAzureDash from 'images/infrastructure_screenshot-crop_journey-guid
 
 import journeyAlertsAndDash from 'images/dashboards_screenshot-crop_journey-dash-and-alerts.png'
 
-Setting up monitors for your data is the foundation of observability, but it's only half of the equation. You need to be able to visualize your data, group streams together, and be notified when your system runs into issues that negatively affect your users. You need to be able to do all this from a centralized location, and be able to adapt to the changing data needs of your organization by curating new visualizations on-the-fly. 
+Setting up monitors for your data is the foundation of observability, but it's only half of the equation. You need to be able to visualize your data, group data points together, and be notified when your system runs into issues. You need to be able to do all this from a centralized location, and be able to adapt to the changing data needs of your organization by creating new visualizations on-the-fly. 
```

Comment:
There's a funny thing with this metaphor. For some reason I think of monitors as being up high, so for me it clashes with this notion of them being a foundation. Just a passing thought...

Reply:
Got something in that's a little more neutral. Thank you!